tests/Model/Permissions/ModelAssetPermissionsTest.php

```diff
@@ -725,23 +725,23 @@ public function testManyElementSearch()
         //update role
         $role = User\Role::getByName('Testrole');
         $role->setWorkspacesAsset([
-            (new User\Workspace\Asset())->setValues(['cId' => $manyElementX->getId(), 'cPath' => $manyElementX->getFullpath(), 'list' => true, 'view' => true]),
-            (new User\Workspace\Asset())->setValues(['cId' => $this->groupfolder->getId(), 'cPath' => $this->groupfolder->getFullpath(), 'list' => true, 'view' => true]),
+            (new User\Workspace\Asset())->setValues(['cId' => $manyElementX->getId(), 'cPath' => $manyElementX->getRealFullPath(), 'list' => true, 'view' => true]),
```

Comment:
part of that PR?

Reply:
Not really....this is required for the tests to pass. when using `getFullpath()` the path is generated per site for frontend request and it fails when no request is available and we just want to test the permissions in the test.